x-pack/plugins/timelines/public/components/t_grid/body/control_columns/checkbox.test.tsx

```diff
@@ -42,11 +42,9 @@ describe('checkbox control column', () => {
 
     test('calls onRowSelected when checked', () => {
       const onRowSelected = jest.fn();
-      const { getByTestId } = render(
-        <RowCheckBox {...defaultProps} onRowSelected={onRowSelected} />
-      );
-
-      fireEvent.click(getByTestId('select-event'));
+      const { container } = render(<RowCheckBox {...defaultProps} onRowSelected={onRowSelected} />);
+      const checkbox = container.querySelector('[data-test-subj~="select-event"]');
```

Comment:
@weltenwort This test was failing after the change I did to include multiple space separated values in the `data-test-subj` attribute and that's how I fixed in the end according to what I found here I checked https://testing-library.com/docs/queries/bytestid.

Then to satisfy Typescript with the container.querySelector
error I used the non-null assertion operator as suggested here https://github.com/testing-library/react-testing-library/issues/119#issuecomment-398898516

Let me know what you think regarding my fix

Reply:
How about this, which fails automatically when the element isn't found and doesn't need a non-null assertion?

```typescript
      const { getByTestId } = render(
        <RowCheckBox {...defaultProps} onRowSelected={onRowSelected} />
      );
      const checkbox = getByTestId(/^select-event/);
      fireEvent.click(checkbox);

```

Reply:
Ok will try it. Thanks for the suggestion!